packages/SwingSet/src/kernel/kernelSyscall.js

```diff
@@ -88,63 +86,53 @@ export function makeKernelSyscallHandler(tools) {
-   * until undefined is returned.  While the initial value for `priorKey` is
-   * conventionally the empty string, in fact any value that is less than
-   * `lowerBound` may be used to the same effect.
+   *   Iteration is accomplished by repeatedly calling `vatstoreGetAfter` in a
+   *   loop, each time pass the key that was returned in the previous iteration,
+   *   until undefined is returned. While the initial value for `priorKey` is
+   *   conventionally the empty string, in fact any value that is less than
+   *   `lowerBound` may be used to the same effect.
    *
-   * Keys in the vatstore are arbitrary strings, but subsets of the keyspace are
-   * often organized hierarchically.  This iteration API allows simple iteration
-   * over an explicit key range or iteration over the set of keys with a given
-   * prefix, depending on how you use it:
+   *   Keys in the vatstore are arbitrary strings, but subsets of the keyspace are
+   *   often organized hierarchically. This iteration API allows simple
+   *   iteration over an explicit key range or iteration over the set of keys
+   *   with a given prefix, depending on how you use it:
    *
-   * Explicitly providing both a lower and an upper bound will enable iteration
-   * over the key range `lowerBound` <= key < `upperBound`
+   *   Explicitly providing both a lower and an upper bound will enable iteration
+   *   over the key range `lowerBound` <= key < `upperBound`
    *
-   * Providing only the lower bound while letting the upper bound default will
-   * iterate over all keys that have `lowerBound` as a prefix.
+   *   Providing only the lower bound while letting the upper bound default will
+   *   iterate over all keys that have `lowerBound` as a prefix.
    *
-   * For example, if the stored keys are:
-   * bar
-   * baz
-   * foocount
-   * foopriority
-   * joober
-   * plugh.3
-   * plugh.47
-   * plugh.8
-   * zot
+   *   For example, if the stored keys are: bar baz foocount foopriority joober
+   *   plugh.3 plugh.47 plugh.8 zot
    *
-   * Then the bounds    would iterate over the key sequence
-   * ----------------   -----------------------------------
-   * 'bar', 'goomba'    bar, baz, foocount, foopriority
-   * 'bar', 'joober'    bar, baz, foocount, foopriority
-   * 'foo'              foocount, foopriority
-   * 'plugh.'           plugh.3, plugh.47, plugh.8
-   * 'baz', 'plugh'     baz, foocount, foopriority, joober
-   * 'bar', '~'         bar, baz, foocount, foopriority, joober, plugh.3, plugh.47, plugh.8, zot
+   *   Then the bounds would iterate over the key sequence
+   *
+   *   'bar', 'goomba' bar, baz, foocount, foopriority 'bar', 'joober' bar, baz,
+   *   foocount, foopriority 'foo' foocount, foopriority 'plugh.' plugh.3,
+   *   plugh.47, plugh.8 'baz', 'plugh' baz, foocount, foopriority, joober
+   *   'bar', '~' bar, baz, foocount, foopriority, joober, plugh.3, plugh.47,
+   *   plugh.8, zot
```

Comment:
Completely unacceptable mangling here.

Reply:
I think this example actually proves the worth of the auto-formatter. Without it, the writer of the comment thought they were writing something intelligible but by JSDoc syntax this is how it appears in an IDE:
<img width="660" alt="Screen Shot 2022-03-23 at 12 54 44 PM" src="https://user-images.githubusercontent.com/21505/159785305-c8705287-41cc-4ca1-a6c0-cdf36656ab5e.png">

If we use conformant JSDoc syntax then it renders something readable:

<img width="851" alt="Screen Shot 2022-03-23 at 12 57 54 PM" src="https://user-images.githubusercontent.com/21505/159785507-bffde094-2e8f-4d21-993f-e0d34b07e454.png">

And the auto-formatter maintains that readability.

So I contend it's a feature that the auto-formatter show the author how JSDoc is interpreting what they're written.

Reply:
+1 I was wondering if there was some appropriate syntax for these cases, also maybe there's a way to make the formatter ignore it in rare cases?

It would make the migration more involved to go back and change all these things but it seems like it would pay off.

Reply:
If JSDoc can't display a simple comment the way I wrote it I have issues with that too.  I really quite unhappy with the slippery slope we've gotten onto, where accessory tooling has increasingly become the tail that wags the dog.

Reply:
This tool/tail is not wagging us. It's a value to make our codebase more clear and our development more efficient. We can add new options to it or if somehow our needs are so different from all the other users we can fork it.

It seems you're taking exception with JSDoc itself. JSDoc is not just code comments; it is a structured syntax. It has already been adopted so I don't think we should spend time debating whether to use it.

Reply:
@FUDCo , I am convinced by the view from the IDE. Preserving the look in the source code for text that loses its meaning in the IDE is not a feature.

@turadg , How did you fix the source so that it renders correctly? Please revise this PR with that fix. Thanks.

For almost all cases, I find the reformatted form a definite improvement. I would like to see this move forward.

Reply:
> How did you fix the source so that it renders correctly?

For the pre-formatted I used a [fenced code block](https://www.markdownguide.org/extended-syntax/#syntax-highlighting)" backticks.  I used "```md" as a hint to the renderer (without it VSCode highlights as JS). Though I discovered that without a qualifier on the code block's syntax (e.g. `md`) the formatter will convert to indentation ([example](https://github.com/Agoric/agoric-sdk/commit/c43b982cba82928eadfa5d18219d2284f29382cd)), which has a stability bug. https://github.com/hosseinmd/prettier-plugin-jsdoc/issues/155

For list items I used Markdown ` - `.

> Please revise this PR with that fix. Thanks.

Done, 6a25dfb38c0b0526d921ad7f95ac1c327f8c5a25